test/contract.js

```diff
@@ -3031,7 +3031,7 @@ describe('typical usage', function() {
                 chainId: '0x1',
                 nonce: '0x1',
             }).then(function (tx) {
-                const expected = tx.rawTransaction;
```

Comment:
Can we also check that this one works, instead of only removing?

Reply:
@cgewecke rawTransaction still exists and prints a deprecation message as commented above.